wallet-new/server/Cardano/Wallet/API/V1/Swagger.hs

```diff
@@ -255,38 +257,63 @@ Software Version   | Git Revision
 
-Please note that wallet web API uses TLS for secure communication. Requests to the API need to send a client CA certificate that was used when launching the node and identifies the client as being permitted to invoke the server API.
+Please note that wallet web API uses TLS for secure communication. Requests to the API need to
+send a client CA certificate that was used when launching the node and identifies the client as
+being permitted to invoke the server API.
 
 Creating a New Wallet
 ---------------------
 
-You can create your first wallet using the `POST /api/v1/wallets` endpoint as follow:
+You can create your first wallet using the [`POST /api/v1/wallets`](#tag/Wallets%2Fpaths%2F~1api~1v1~1wallets%2Fpost) endpoint as follow:
 
 ```
-curl -X POST https://localhost:8090/api/v1/wallets                     \
-     -H "Content-Type: application/json; charset=utf-8"                \
-     -H "Accept: application/json; charset=utf-8"                      \
-     --cacert ./scripts/tls-files/ca.crt                               \
-     -d '{                                                             \
-  "operation": "create",                                               \
-  "backupPhrase": ["squirrel", "material", "silly", "twice", "direct", \
-    "slush", "pistol", "razor", "become", "junk", "kingdom", "flee"],  \
-  "assuranceLevel": "normal",                                          \
-  "name": "MyFirstWallet"                                              \
+curl -X POST https://localhost:8090/api/v1/wallets \
+  -H "Accept: application/json; charset=utf-8" \
+  -H "Content-Type: application/json; charset=utf-8" \
+  --cacert ./scripts/tls-files/ca.crt \
+  -d '{
+  "operation": "create",
+  "backupPhrase": ["squirrel", "material", "silly", "twice", "direct",
+    "slush", "pistol", "razor", "become", "junk", "kingdom", "flee"],
+  "assuranceLevel": "normal",
+  "name": "MyFirstWallet",
+  "spendingPassword": "5416b2988745725998907addf4613c9b0764f04959030e1b81c603b920a115d0"
 }'
 ```
 
-> **Warning**: Those 12 mnemonic words given for the backup phrase act as an example. **Do not** use them on a production system. See the section below about mnemonic codes for more information.
+> **Warning**: Those 12 mnemonic words given for the backup phrase act as an example. **Do
+> not** use them on a production system. See the section below about mnemonic codes for more
+> information.
 
-As a response, the API provides you with a unique wallet `id` to be used in subsequent requests. Make sure to store it / write it down. Note that every API response is [jsend-compliant](https://labs.omniti.com/labs/jsend); Cardano also augments responses with meta-data specific to pagination. More details in the section below about *Pagination*.
+The `spendingPassword` is optional but highly recommended. It is an original UTF-8
+string of 32 characters, encoded in base 16, yielding to an hexadecimal sequence of 64 bytes.
```

Comment:
What's the meaning of a UTF-8 encoded *and* base 16 encoded string?

Reply:
We read from a utf-8 source and convert that to an hexadecimal sequence. Actually, the input can be pretty much anything, it could be latin-1 encoded initially or ANSI.. Doesn't matter much and perhaps it is confusing here. We'll always be manipulating the hex output anyway.